### DISCUSSION TITLE
Hydra CLI - Create a Client with owner and metadata info

### DISCUSSION THREAD

**Discussion by pommelinho** [ory/hydra#2432] [Q&A] [answered]
Hey!

Accoding to https://www.ory.sh/hydra/docs/cli/hydra-clients-create it is not possible to add owner or metadata info via CLI when creating in client?

Is it planned to support this?

**Comment by vinckr** [comment]
I think so yes, I could not find the respective issue at the moment.

**Reply by aeneasr** [reply]
Contribs welcomed!

**Comment by vinckr** [comment]
If this feature requests interests you, feel free to open an issue with more concrete implementation ideas, 
otherwise it can be further discussed in here, moving it to `ideas`.

**Comment by apexskier** [comment, marked as answer]
It's possible to set metadata for a client using the admin API:

```sh
curl -X POST "$HYDRA_URL/clients" -H 'Content-Type: application/json' --data-raw '{
  "client_id": "auth-oidc-test-client",
  "client_name": "OIDC Test Client",
  "client_secret": "$CLIENT_SECRET",
  "redirect_uris": [
    "http://auth-oidc-test-client.test/callback",
  ],
  "grant_types": [
    "authorization_code"
  ],
  "response_types": [
    "code"
  ],
  "scope": "offline_access offline openid",
  "metadata": {
    "my_metadata": "value"
  }
}'
```
